src/utils/gtags.client.ts

```diff
@@ -51,11 +53,18 @@ export const event = ({
     )
     return
   }
-  window.gtag("event", action, {
+
+  const eventPayload: Record<string, any> = {
     event_category: category,
     event_label: label,
     value: value,
-  })
+  };
+
+  if (location) {
+    eventPayload.page_location = location;
```

Comment:
What if instead of grabbing `window.location.hostname` outside of this and passing it in you made `location?` a bool as to grab it yes or no, then just get the `window.location.hostname` inside of this function if true? Then you don't have to repeat the code outside of this so much.